.github/workflows/ci.yml

```diff
@@ -25,8 +25,20 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r backend/requirements.txt
+    - name: set up docker compose
+      run: |
+        mkdir env
+        echo "MONGODB_USERNAME=github-ci" >> env/backend.env
+        echo "MONGODB_PASSWORD=github-ci" >> env/backend.env
+        echo "MONGODB_DATABASE=mongodb" >> env/backend.env
+        echo "MONGO_INITDB_ROOT_USERNAME=github-ci" >> env/mongodb.env
+        echo "MONGO_INITDB_ROOT_PASSWORD=github-ci" >> env/mongodb.env
+        docker compose build backend
+        docker compose up backend -d
```

Comment:
Nice update to the workflow! GitHub Actions is pog. You could combine lines 36 and 37 with a --build on 37 to force build on the compose but makes no difference otherwise 🙂

Reply:
ooh i didnt know about this param - changed 😃